### ISSUE TITLE
Please, use exact same name for countries and region across your datasets

### ISSUE THREAD

**Issue by rosericazondekon** [CSSEGISandData/COVID-19#423]
Please, report data accurately, stop using inconsistent country names. For example, either use Iran or "Iran (Islamic Republic of)", "South Korea" or "Republic of Korea", but at least, be consistent. "Taipei and environs" is not a country, Taiwan is (and not a Province/State). Many people are building visualizations systems on top of your data, so please, help us by being consistent.
Thank you for the awesome work!

**Comment by sibblegp** [comment]
#420 

What have you built?

**Comment by rosericazondekon** [comment]
> #420
> 
> What have you built?

I built a near real-time interactive visualization tool with these data.

**Comment by sibblegp** [comment]
Cool. Is it public? Would love to see.

**Comment by rosericazondekon** [comment]
> Cool. Is it public? Would love to see.

I would love to make it public and share it with the world, but I do not have permission to share it at this moment. My leadership hasn't approved it.

**Comment by daniel-farkas** [comment]
@sibblegp Here's ours, in case you are interested: https://datoris.com/gallery/report/92

**Comment by rosericazondekon** [comment]
> @sibblegp Here's ours, in case you are interested: https://datoris.com/gallery/report/92

Awesome job on your dashboard. I really wish I could share mine too.

**Comment by 3verse** [comment]
> @sibblegp Here's ours, in case you are interested: https://datoris.com/gallery/report/92

Thanks for sharing Daniel it's a very interesting dashboard especially the fatality rate over time, I've been working on something similar based on daily data: https://public.tableau.com/profile/salvatore1114#!/vizhome/wdctest/CoronavirusCOVID-19Dashboard-DailyUpdates - feel free to let me know if you have any suggestions or comments.

I'm having the same issues with inconsistent data coming through, hopefully will be resolved soon. Later tonight I'll need to create a "calculated field" to make up for the inconsistencies in the meantime

**Comment by MerliMomo** [comment]
Agreed about the country names. Today it is Korea, South. I have also built visualisation in Power BI with a timelapse visual like watching 'Plague'. Loving your work otherwise.
